### DISCUSSION TITLE
关于字体问题的临时方案

### DISCUSSION THREAD

**Discussion by yzx23333** [chinosk6/gakuen-imas-localify#11] [Ideas]
个人~~观察~~了下游戏，使用了游戏中的两种字体IBMPlexSansJP-SemiBold+Campus-Alphanumeric-SemiBold然后和MO-UDShinGo-SC-Gb4-M混合了一下做出了这个
应该可以缓解框框和部分字母数字错位的问题，也许可以作为临时解决方案
感觉pr不是很合适看能不能发在这里~~以方便撤回~~
[gkamsZHFontMIX.zip](https://github.com/user-attachments/files/15543138/gkamsZHFontMIX.zip)

**Comment by chinosk6** [comment]
经过测试可以使用，效果不错，感谢大佬的贡献~
您愿意的话欢迎去 [资源仓库](https://github.com/chinosk6/GakumasTranslationData) 提交 PR

**Reply by yzx23333** [reply]
好的，那么字体文件的文件名有要求吗

**Reply by chinosk6** [reply]
你喜欢就行，主仓库这边的路径会根据你的命名来改的~

**Comment by yzx23333** [comment]
看起来可以先close了
https://github.com/chinosk6/gakuen-imas-localify/commit/d7ff38b3f0336eb7aaf5dc01e34c967aaa5be8c2
